modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -52,6 +53,45 @@ It will be updated to say `Node property uniqueness constraints added: 1` in Neo
 ======
 
 
+[discrete]
+[[constraints-create-a-node-uniqueness-constraint-by-param]]
+=== Create a node property uniqueness constraint using a parameter
+
+The constraint name can also be given as a parameter.
+
+.+CREATE CONSTRAINT+
+======
+
+.Parameters
+[source,javascript, indent=0]
+----
+{
+  "name": "node_uniqueness_param"
+}
+----
+
+.Query
+[source, cypher]
+----
+CREATE CONSTRAINT $name
+FOR (book:Book) REQUIRE book.prop1 IS UNIQUE
+----
+
+.Result
+[queryresult]
+----
+Added 1 constraint.
+----
+
+[NOTE]
+====
+The detailed statistics view currently says `Unique constraints added: 1`.
+It will be updated to say `Node property uniqueness constraints added: 1` in Neo4j version 6.0.
```

Comment:
The comment was added cause it was decided it was to breaking to just update (not sure if all it included the statistics messages as well or just all other places but it was decided to not update any of them now and do all of them later since at least most places (like index type and error messages) were definitely considered breaking.

I think that when the comments were first added the examples actually printed the original string, but they have since then been updated not to, which makes the comments feel like they miss context 🤷